cernopendata/modules/fixtures/data/docs/atlas-about/atlas-about.md

```diff
@@ -1,7 +1,54 @@
-ATLAS (A Toroidal LHC ApparatuS) is one of the two general-purpose detectors at the LHC, the other being CMS. Both detectors are geared towards covering a large range of physics produced in proton collisions such as the search for the Higgs Boson or Vector Boson Scattering.
+**[ATLAS](https://atlas.cern)** (A Toroidal LHC ApparatuS) is one of the four major experiments at the [Large Hadron Collider](http://home.web.cern.ch/topics/large-hadron-collider) (LHC) at [CERN](https://home.cern). It is a general-purpose particle physics experiment run by an international collaboration and, together with [ATLAS](https://atlas.cern), is designed to exploit the full discovery potential and the vast range of physics opportunities that the LHC provides.
```

Comment:
Need to change the text to `[CMS](https://cms.cern)`.

Reply:
Thank you for spotting this.
Done.